uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/PublicHolidayLeaveRequestCreationTest.php

```diff
@@ -101,4 +102,17 @@ public function testCanCreatePublicHolidayLeaveRequestsForAnAbsenceType() {
 
     $this->assertEquals(-2, LeaveBalanceChange::getLeaveRequestBalanceForEntitlement($periodEntitlement));
   }
+
+  /**
+   * @expectedException InvalidArgumentException
+   * @expectedExceptionMessage It's not possible to create Public Holidays for Absence Types where 'Must take public holiday as leave' is false
```

Comment:
if you added a ts to the exception above you may need to use **expectedExceptionCode** instead **expectedExceptionMessage** , but make sure to add a code when you through the exception